iidm/iidm-api/src/main/java/com/powsybl/iidm/network/DcTerminal.java

```diff
@@ -86,4 +90,38 @@ public interface DcTerminal {
      * @see VariantManager
      */
     DcTerminal setI(double i);
+
+    void traverse(TopologyTraverser traverser);
```

Comment:
Could you add the method with the `TraversaleType` parameter, so that these DcTerminal follow the same structures as the AC Terminals?

```suggestion
    /**
     * Traverse the DC topology graph.
     * @param traverser traversal handler
     */
    void traverse(TopologyTraverser traverser);

    /**
     * Traverse the DC topology graph.
     * @param traverser traversal handler
     * @param traversalType traversal type
     */
    void traverse(TopologyTraverser traverser, TraversalType traversalType);
```